src/components/Prompt/Prompt.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+@keyframes Prompt_open {
+    0% {
+        opacity: 0;
+        transform: translateY(100%);
+    }
+    100% {
+        opacity: 1;
+        transform: translateY(0%);
+    }
+}
+
+@keyframes Prompt_close {
```

Comment:
let's name it in the way other projects do https://github.com/gravity-ui/uikit/blob/6fc04590dabcc0e968ebdd0d13fedf6d0b6d1bc9/src/components/Popup/Popup.scss#L167